ossfuzz/fuzz_data_producer.h

```diff
@@ -3,8 +3,22 @@
 #include <stdio.h>
 #include <stdlib.h>
 
-FUZZ_STATIC uint32_t FUZZ_produceUint32Range(uint8_t *data, size_t size,
-                                             uint32_t min, uint32_t max) {
+typedef struct {
+  const uint8_t *data;
+  size_t size;
+} FUZZ_dataProducer_t;
+
+FUZZ_dataProducer_t *FUZZ_dataProducer_create(const uint8_t *data, size_t size) {
+  FUZZ_dataProducer_t *producer = malloc(sizeof(FUZZ_dataProducer_t));
+  producer->data = data;
+  producer->size = size;
+  return producer;
+}
+
+void FUZZ_dataProducer_free(FUZZ_dataProducer_t *producer) { free(producer); }
+
+uint32_t FUZZ_dataProducer_uint32(FUZZ_dataProducer_t *producer, uint32_t min,
+                                  uint32_t max) {
   if (min > max) {
```

Comment:
Should I be using an assert instead?

Reply:
Use `FUZZ_ASSERT` from `fuzz_helpers.h`. We want to make sure that we still crash even if `NDEBUG` is defined.